drivers/gpio/gpio_mchp_xec.c

```diff
@@ -137,9 +144,6 @@ static int gpio_xec_configure(const struct device *dev,
 		/* Already configured */
```

Comment:
@scottwcpg Maybe worth removing elseif and else since not  doing anything anymore.

Reply:
@albertofloyd Pushed rebase removing the empty else if and else.